model/constants.py

```diff
@@ -9,3 +9,4 @@
 epochs_per_month = 6750
 epochs_per_year = 82180
 pow_blocks_per_epoch = 32.0 * 12 / 13
+eth_deposited_per_validator = 32
```

Comment:
I suggest using `MAX_EFFECTIVE_BALANCE` from the system parameters file and removing this line.